app/front-end/src/features/editor/components/toolbarView/toolbarGroupsSelector.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, List, useTheme } from '@mui/material';
+
+export interface ToolbarGroupsSelectorProps {
+  children: React.ReactNode;
+}
+
+export const ToolbarGroupsSelector: React.FC<ToolbarGroupsSelectorProps> = ({ children }) => {
+  return <List sx={{ height: '25%', p: '0', overflow: 'auto' }}>{children}</List>;
+};
+
+export interface ToolbarGroupsSelectorItemProps {
+  id: string;
+  label: string;
+  onClick: () => void;
+  groupRef?: string;
+}
```

Comment:
Exports could be seperated into different files

Reply:
Same response